src/builder-source-archive.c

```diff
@@ -677,13 +738,34 @@ builder_source_archive_class_init (BuilderSourceArchiveClass *klass)
                                                         "",
                                                         NULL,
                                                         G_PARAM_READWRITE));
+  g_object_class_install_property (object_class,
```

Comment:
I realise the existing `sha256` property doesn’t have any documentation, but it would be nice to add at least a comment saying what the checksum is a checksum of, for each of these.

Reply:
These are all internal properties just so that the json-glib serialization stuff works. I don't think documenting them here is useful, the useful place is the user facing documentation.